backend/src/bundles/users/user.repository.ts

```diff
@@ -43,6 +43,16 @@ class UserRepository
         return users.map((it) => UserEntity.initialize(it));
     }
 
+    public async updateById(
+        id: string,
+        data: object,
+    ): Promise<UserEntity | null> {
+        await this.model.query().update(data).where('id', id).execute();
+
+        const user = await this.model.query().findOne({ id });
+        return user ? UserEntity.initialize(user) : null;
+    }
```

Comment:
Maria wrote this method, however, it has 0 usage across the app, maybe remove it?